apis/v1alpha1/httproute_types.go

```diff
@@ -19,18 +19,34 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// +genclient
```

Comment:
See https://github.com/kubernetes-sigs/service-apis/pull/336#discussion_r496850613

Reply:
Will save that for #336 if that works for you. This isn't actually an addition, just moving the type to the top.